src/harness/testcases/testdata/pal_record_0.json

```diff
@@ -0,0 +1,28 @@
+{
+  "palId": "pal/05-2017/20041084200/A",
```

Comment:
I think we probably want to only fill in the fields that will *not* be replaced so that it's clear what the behavior is. What do you think?